packages/template-retail-react-app/babel.config.js

```diff
@@ -4,4 +4,9 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-module.exports = require('pwa-kit-cli/configs/babel/babel-config')
+
+// NOTE: THIS IS AN EXAMPLE OF HOW TO USE A CUSTOM BABEL CONFIGURATION. SIMPLY DEFINE A BABEL.CONFIG.JS
+// AND USE THE PWA-KIT PRESET ADDING ANY ADDITIONAL PLUGINS OR CHANGES NECESSARY.
```

Comment:
```suggestion
// NOTE: THIS IS AN EXAMPLE OF HOW TO USE A CUSTOM BABEL CONFIGURATION. SIMPLY DEFINE A BABEL.CONFIG.JS
// AND USE THE PWA-KIT PRESET, ADDING ANY ADDITIONAL PLUGINS OR CHANGES NECESSARY.
```
Mostly the comma, but also all caps is rather aggressive.
```suggestion
// Note: This is an example of how to use a custom babel configuration. Simply define a babel.config.js
// and use the pwa-kit preset, adding any additional plugins or changes necessary.
```

Reply:
This isn't something that I'm planning on leaving in the code. It was solely for demo purposes.